packages/graz/src/actions/account.ts

```diff
@@ -24,6 +25,13 @@ export const connect = async (args?: ConnectArgs): Promise<ConnectResult> => {
   try {
     const { recentChainIds: recentChains, chains, walletType } = useGrazInternalStore.getState();
 
+    const walletConnectInstance = getWallet(WalletType.WALLETCONNECT);
+    const { disable: walletConnectDisable } = walletConnectInstance;
+
+    if (walletConnectDisable) {
+      void walletConnectDisable();
+    }
```

Comment:
you need to wrap this if the wallet type they are trying to connect with is wallet connect